omnibus/package-scripts/agent/prerm

```diff
@@ -161,8 +161,8 @@ else
 fi
 
 # Delete all .pyc files in the `agent/` and the `bin/agent/dist` dirs
-find $INSTALL_DIR/agent -name '*.py[co]' -type f -delete || echo "Unable to delete .pyc files in $INSTALL_DIR/agent"
-find $INSTALL_DIR/bin/agent/dist -name '*.py[co]' -type f -delete || echo "Unable to delete .pyc files in $INSTALL_DIR/bin/agent/dist"
-find $INSTALL_DIR/bin/agent/dist -name '__pycache__' -type d -delete || echo "Unable to delete __pycache__ directories in $INSTALL_DIR/bin/agent/dist"
+find $INSTALL_DIR/embedded/lib/python* -name '*.py[co]' -type f -delete || true
+find $INSTALL_DIR/bin/agent/dist -name '*.py[co]' -type f -delete || true
+find $INSTALL_DIR/bin/agent/dist -name '__pycache__' -type d -delete || true
```

Comment:
might be fine to keep the error messages here? (it's only shown if `$INSTALL_DIR/bin/agent/dist` doesn't exist, and in that case it's good to print a warning, right?)